src/main/java/org/sagebionetworks/bridge/S3Initializer.java

```diff
@@ -0,0 +1,148 @@
+            + "                \"s3:GetBucketLocation\""
+            + "            ],"
+            + "            \"Resource\": \"arn:aws:s3:::${bucketName}\""
+            + "        },"
+            + "        {"
+            + "            \"Effect\": \"Allow\","
+            + "            \"Principal\": {"
+            + "                \"AWS\": \"arn:aws:iam::325565585839:root\""
+            + "            },"
+            + "            \"Action\": ["
+            + "                \"s3:GetObject*\","
+            + "                \"s3:*MultipartUpload*\""
+            + "            ],"
+            + "            \"Resource\": \"arn:aws:s3:::${bucketName}/*\""
+            + "        }"
+            + "    ]"
+            + "}";
+    
+    private static final String PUBLIC_ACCESS_POLICY = "{"
+            + "    \"Version\": \"2012-10-17\","
+            + "    \"Id\": \"Policy20160803001\","
+            + "    \"Statement\": ["
+            + "        {"
+            + "            \"Sid\": \"Stmt20160803001\","
+            + "            \"Effect\": \"Allow\","
+            + "            \"Principal\": \"*\","
+            + "            \"Action\": \"s3:GetObject\","
+            + "            \"Resource\": \"arn:aws:s3:::${bucketName}/*\""
+            + "        }"
+            + "    ]"
+            + "}";
+    
+    private static final BucketCrossOriginConfiguration ALLOW_PUT = new BucketCrossOriginConfiguration()
+            .withRules(new CORSRule()
+                    .withAllowedHeaders(ImmutableList.of("*"))
+                    .withAllowedMethods(ImmutableList.of(AllowedMethods.PUT))
+                    .withAllowedOrigins(ImmutableList.of("*"))
+                    .withMaxAgeSeconds(3000));
+    
+    public static enum BucketType {
+        INTERNAL(null, null),
+        INTERNAL_UPLOAD_ACCESSIBLE(null, ALLOW_PUT),
+        SYNAPSE_ACCESSIBLE(SYNAPSE_ACCESS_POLICY, null),
+        PUBLIC_ACCESSIBLE(PUBLIC_ACCESS_POLICY, null);
+        
+        String policy;
+        BucketCrossOriginConfiguration corsConfig;
+        private BucketType(String policy, BucketCrossOriginConfiguration corsConfig) {
+            this.policy = policy;
+            this.corsConfig = corsConfig;
+        }
+    }
+    
+    private BridgeConfig bridgeConfig;
+    
+    private AmazonS3Client s3Client;
+    
+    @Autowired
+    public final void setBridgeConfig(BridgeConfig bridgeConfig) {
+        this.bridgeConfig = bridgeConfig;
+    }
+    
+    @Autowired
+    public final void setS3Client(AmazonS3Client s3Client) {
+        this.s3Client = s3Client;
+    }
+    
+    private static Map<String, BucketType> S3_BUCKET_PROP_NAMES = new ImmutableMap.Builder<String, BucketType>()
+            .put("attachment.bucket", BucketType.SYNAPSE_ACCESSIBLE)
+            .put("health.data.bucket.raw", BucketType.SYNAPSE_ACCESSIBLE)
+            .put("health.data.bucket.processed", BucketType.SYNAPSE_ACCESSIBLE)
```

Comment:
Similarly, we can not worry about these for now and migrate them at a later date.